### PR TITLE
Trackers: jsb000.tistory.com

### PR DESCRIPTION
**Pull request by ghost**
### URL: `https://jsb000.tistory.com/1695`
Screenshot:
Screenshot

### Network Logs
network
network2

### Code Preview `https://t1.daumcdn.net/tiara/js/v1/tiara.min.js`
codepreview

### CODE REVIEW

File: SpywareFilter/sections/tracking_servers.txt [hunk 1]
```diff
@@ -38,6 +38,16 @@
 !||i.snssdk.com/log/*
 !||i.snssdk.com/slardar/sdk.js
 !
+||stat.tiara.tistory.com^
```
Comment (Alex-302) [inline SpywareFilter/sections/tracking_servers.txt:41]: Can be added as
```
||stat.tiara.
||track.tiara.
```
They still will be blocked by DNS and ad blockers.
image

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: SpywareFilter/sections/tracking_servers.txt (hunk 1 above)
Comment (piquark6046) [reply SpywareFilter/sections/tracking_servers.txt:41]: OK.

Comment (piquark6046) [reply SpywareFilter/sections/tracking_servers.txt]: 980bcee91d1d5d90498429e158d1c51164bcac81

**Review by zloyden — APPROVED** [review]

**Comment by piquark6046** [comment]
@Alex-302 Cloud you merge this PR?

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**